content/docs/18-knowledgebase/00-how-to/01-how-to-setup-kube-api-audit-logging.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "How to Configure kube-apiserver audit logging"
+metaTitle: "Configuring the kube-apiserver audit logging featureo"
+metaDescription: "KnowledgeBase with How to"
+icon: "laptop"
+hideToC: true
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Getting Started with Palette and Auditing Your Kubernetes Cluster Using kube-apiserver Audit Logs
+
+This tutorial is based on the upstream Kubernetes documentation and kube-apiserver source code. It assumes that you have working knowledge of the Kubernetes API.
+
+## What does kubernetes audit logging offer?
+
+Kubernetes auditing provides a security-relevant, chronological set of records documenting the sequence of actions in a cluster. The cluster audits the activities generated by users, by applications that use the Kubernetes API, and by the control plane itself.
+
+Auditing allows cluster administrators to answer the following questions:
+  - what happened?
+  - when did it happen?
+  - who initiated it?
+  - on what did it happen?
+  - where was it observed?
+  - from where was it initiated?
+  - to where was it going?
+
+## What are some scenarios where you would need to enable kube-api audit logging?
+
+- to meet internal security controls
+- to ensure that the actions of individual users can be uniquely traced back to a specific user
+- to debug an issue where resources are being modified by an unknown application
+
+
+## Performance considerations when enabling audit logging
+
+The audit logging feature increases the memory consumption of the API server because some context required for auditing is stored for each request. Memory consumption depends on the audit logging configuration.
+
+
+## Logging all API requests at the metadata level
+
+To start, we will use the [minimal audit policy file](#creating-the-audit-policy-file) to log all requests at the `metadata` level. 
```

Comment:
```suggestion
To start, you will use the [minimal audit policy file](#creating-the-audit-policy-file) to log all requests at the `metadata` level. 
```
Address the user through "you". Only use "we" when providing a recommendation our making a statement from us as a company.